with-formdata-image-upload/README.md

```diff
@@ -1,18 +1,20 @@
 # Image Upload Example
 
-Try it at https://expo.io/@community/image-upload-example
+Try it at https://expo.io/@documentation/imageUploadExample
 
 ## How to use
 
+- Use [this link](https://snack.expo.io/@documentation/image-upload-example) to open this project in your browser using Snack
```

Comment:
The last link wasn't a saved snack, so I just made one on the documentation account, and also linked to the published version above